src/meshapi/tests/test_replay_join_records.py

```diff
@@ -0,0 +1,145 @@
+from unittest.mock import patch
+
+from django.core import management
+from django.test import TestCase
+
+from meshapi.models.install import Install
+from meshapi.util.join_records import JoinRecord, JoinRecordProcessor, s3_content_to_join_record
+
+MOCK_JOIN_RECORD_PREFIX = "join-record-test"
+
+
+# Integration test to ensure that we can fetch JoinRecords from an S3 bucket,
+# replay them into the join form, and insert the objects we expect.
+# We should test a happy case, a case when the JoinRecord is bad (like a JoinRecord
+# somehow from Russia), and maybe mock a MeshDB 500 that just ends in us putting
+# the data back.
+class TestReplayJoinRecords(TestCase):
+    def setUp(self) -> None:
+        p = JoinRecordProcessor()
```

Comment:
nit: if you made this an instance variable you wouldn't need to construct so many of them